include/s3select_functions.h

```diff
@@ -26,7 +26,7 @@ struct push_4dig
 };
 static push_4dig g_push_4dig;
 
-enum class s3select_func_En_t {ADD,SUM,MIN,MAX,COUNT,TO_INT,TO_FLOAT,TO_TIMESTAMP,SUBSTR,EXTRACT,DATE_ADD,DATE_DIFF,UTCNOW};
+enum class s3select_func_En_t {ADD,SUM,AVG,MIN,MAX,COUNT,TO_INT,TO_FLOAT,TO_TIMESTAMP,SUBSTR,EXTRACT,DATE_ADD,DATE_DIFF,UTCNOW};
```

Comment:
please break across multiple lines:
enum class s3select_func_En_t {
ADD,
SUM,
...
(I think Gal's respacing commit will do this for all the code, but don't restore it)

Reply:
Modified Matt. Thanks for reviewing